tasmota/tasmota_template_legacy.h

```diff
@@ -190,6 +190,7 @@ enum LegacyUserSelectablePins {
   GPI8_OLED_RESET,     // OLED Display Reset
   GPI8_SOLAXX1_TX,     // Solax Inverter tx pin
   GPI8_SOLAXX1_RX,     // Solax Inverter rx pin
+  GPI8_SOLAXX1_RTS,    // Solax Inverter RTS pin
```

Comment:
Thanks for your PR. You can't just insert new GPIO codes in the existing list, as it would break any existing template. You need to add new GPIOs at the end of the list.

Reply:
In addition, no changes to tasmota_template_legacy.h supported. Won't be used as there is no legacy code using it.

Pls remove any changes here.